stable/wordpress/README.md

```diff
@@ -86,6 +86,7 @@ The following table lists the configurable parameters of the WordPress chart and
 | `ingress.hosts[0].tls`               | Utilize TLS backend in ingress             | `false`                                                    |
 | `ingress.hosts[0].tlsSecret`         | TLS Secret (certificates)                  | `wordpress.local-tls-secret`                               |
 | `ingress.hosts[0].annotations`       | Annotations for this host's ingress record | `[]`                                                       |
+| `ingress.hosts[0].labels`            | Labels for this host's ingress records     | `{}`                                                       |
```

Comment:
I'd state very clear that they are "extra labels". You're not modifying the default ones:
```
     app:
     chart:
     release
     heritage:
```

Reply:
We're seemingly converging to use `extraLabels` for such keys. Like `extraAnnotations` and `extraEnv`, etc. I'd recommend renaming this key to `ingress.hosts[0].extraLabels` as well.

Reply:
Do you think I can rename 'annotations' in the same PR (this is a breaking change).